accepted/00000-use-dnf-for-rpm-repos.md

```diff
@@ -0,0 +1,177 @@
+- Feature Name: Use DNF for managing the RPM content
+- Start Date: 04-04-2019
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Uyuni contains components that support importing external repositories;
+collecting and retrieving information on packages, products, modules, and other content;
+and hosting repositories to be consumed by Spacewalk clients. For RPM repositories,
+it currently uses Zypper and libsolv (and previously used YUM), per
+[a previously implemented RFC](https://github.com/uyuni-project/uyuni-rfc/blob/master/accepted/00047-yum-replacement.md).
+
+However, this makes it difficult to run Uyuni on Red Hat Enterprise Linux and CentOS servers
+without replacing base packages (violating subscription support terms in the RHEL case),
```

Comment:
Yeah. The actual problem is that zypper requires enabling HELIXREPO and SUSEREPO libsolv features. They're not enabled in RHEL/CentOS and they don't want to enable it (I've asked as the Fedora zypper maintainer before).

Zypper team has no interest in fixing it so that those aren't required, since the former is used for the solver testcases, and the latter is the legacy (<SLE10) YaST2 metadata stuff.